PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/DiningAdapter.java

```diff
@@ -126,13 +132,25 @@ public void call(Throwable throwable) {
     private class MenuComparator implements Comparator<DiningHall> {
         @Override
         public int compare(DiningHall diningHall, DiningHall diningHall2) {
```

Comment:
While this Compare method works, it's a bit confusing to follow since the `sortBy` if statements have different logic. You should have one block for `sortBy.equals("OPEN")`, another block for `sortBy.equals("RESIDENTIAL")`, and finally the default `diningHall.getName().compareTo(diningHall2.getName())` at the end. It'd be even better if you used a switch statement for `sortBy`.

Reply:
All changes implemented!